web/tests/lib/test_stream.js

```diff
@@ -0,0 +1,36 @@
+"use strict";
+
+exports.make_stream = (opts = {}) => {
+    const default_stream = {
+        audible_notifications: false,
+        can_remove_subscribers_group: 0,
+        color: "blue",
+        creator_id: null,
+        date_created: Date.now(),
+        description: "A Scandinavian country",
+        desktop_notifications: false,
+        email_address: "test@example.com",
+        email_notifications: false,
+        first_message_id: null,
+        history_public_to_subscribers: true,
+        invite_only: false,
+        is_announcement_only: false,
+        is_muted: false,
+        is_web_public: true,
+        message_retention_days: null,
+        name: "Denmark",
+        newly_subscribed: false,
+        pin_to_top: false,
+        previously_subscribed: false,
+        push_notifications: false,
+        render_subscribers: false,
+        rendered_description: "A Scandinavian country",
+        stream_id: 1,
```

Comment:
One idea that we've found helpful is to generate these default IDs automatically, so that they're always different from each other. See this example:
https://github.com/zulip/zulip-flutter/blob/51e01f416a06be746f03292f41a34bb109a8b914/test/example_data.dart#L280-L294

In particular for messages, a further helpful twist is that the automatic IDs are generated in order. Then once you're doing that for messages, one might as well do the same thing for streams and users and so on.